examples/observability/adot-amp-grafana-for-haproxy/variables.tf

```diff
@@ -10,3 +10,8 @@ variable "grafana_api_key" {
   default     = ""
   sensitive   = true
 }
+
+variable "aws_region" {
+  description = "AWS Region to deploy resources"
+  type        = string
+}
```

Comment:
Provide `default = us-west-2` so our CI doesn't wait for input.

Reply:
Thanks for the review! Can you please merge if that's good for you?